pkg/action/build-uki.go

```diff
@@ -0,0 +1,538 @@
+package action
+
+import (
+	"compress/gzip"
+	"fmt"
+	"io"
+	"math"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+
+	"github.com/u-root/u-root/pkg/cpio"
+	"golang.org/x/exp/maps"
+
+	"github.com/kairos-io/enki/pkg/types"
+	"github.com/kairos-io/kairos-agent/v2/pkg/elemental"
+	v1 "github.com/kairos-io/kairos-agent/v2/pkg/types/v1"
+)
+
+const Cmdline = "console=ttyS0 console=tty1 net.ifnames=1 rd.immucore.oemlabel=COS_OEM rd.immucore.debug rd.immucore.oemtimeout=2 rd.immucore.uki selinux=0"
```

Comment:
this should be user overridable IMHO. Nice to have and we ALWAYS would need to set things like `rd.immucore.oemlabel=COS_OEM rd.immucore.oemtimeout=2 rd.immucore.uki` but the rest can be user set I think